Amazon.IonObjectMapper.Test/NamingConventionTest.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Should these changes be reverted since nothing was done to this file?

Reply:
Reverted.